data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/teleport/tunnel_source_teleport_unit_fail.cfg

```diff
@@ -0,0 +1,71 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [teleport][tunnel]
+##
+# Actions:
+# Give side 1 units the ability to teleport but that has a tunnel defined that uses teleport_unit in its source
```

Comment:
Saying "owned villages" would be clearer here, documentation is better if it conveys the same information without repeating the exact code.

Reply:
Updated.